lib/reviewer/github.rb

```diff
@@ -2,7 +2,7 @@
 require 'graphql/client/http'
 
 module GitHub
-  HTTP = GraphQL::Client::HTTP.new('https://api.github.com/graphql') do
+  HTTP = GraphQL::Client::HTTP.new(ENV['DANGER_GITHUB_GRAPHQL_URL'] || 'https://api.github.com/graphql') do
```

Comment:
`DANGER_GITHUB_GRAPHQL_URL` is a new environment variable not found in https://github.com/danger/danger.
What kind of naming do you think is good?